azerothcore/azerothcore-wotlk
# fix(Core/Creature): Respawning mobs missing aggro "grace period"

## Changes Proposed:
-  creatures will now wait 5 seconds after respawn until they attack, unless they are attacked before grace period fade


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4811


## Tests Performed:
- Tested in-game


## How to Test the Changes:
Case 1:
- Kill a mob
- Wait for it to respawn
- Check if attacks after 5 seconds

Case 2:
- Kill a mob
- Wait for it to respawn
- Try attacking before grace period fades to check if mob will attack you


## Target Branch(es):

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
